### PR TITLE
Restore `pr-filters` and adjust spacing

### PR DESCRIPTION
**Pull request by FloEdelmann**
Currently, the Draft PR filters are not added to the *Reviews* filter, but to the *Assignee* filter in [some repos](https://github.com/OpenLightingProject/open-fixture-library/pulls) and not at all in [others](https://github.com/sindresorhus/refined-github/pulls). The Status filter is not added in either one.

This PR does the following:

* Fix selectors (to show both filters again)
* Use new dropdown syntax (to fix the layout)
* Fix spacing between dropdown buttons
* Cache CI status for faster Status filter (checks another item from #2671)

# Test

* https://github.com/OpenLightingProject/open-fixture-library/pulls
* https://github.com/sindresorhus/refined-github/pulls
* https://github.com/sindresorhus/refined-github/issues (spacing is fixed there, too)

### CODE REVIEW

File: source/features/pr-filters.tsx [hunk 1]
```diff
@@ -64,25 +68,35 @@ async function addStatusFilter(): Promise<void> {
...
-	// TODO: replace this with an API call
-	const hasCI = await fetchCIStatus();
+	const cachedCIStatus = cache.function(async () => {
+		// TODO: replace this with an API call
+		const statusIcon = await fetchCIStatus();
+		return statusIcon !== undefined;
+	}, {
+		expiration: 3,
+		cacheKey: () => __featureName__ + ':' + getRepoURL()
+	});
```
Comment (fregante) [inline source/features/pr-filters.tsx]: This function needs to be moved to the top-level scope. Best yet if we update it to use the API, example:

```gql
{
  repository(owner: "sindresorhus", name: "refined-github") {
    head: object(expression: "HEAD") {
      ... on Commit {
        history(first: 10) {
          nodes {
            status {
              state
            }
          }
        }
      }
    }
  }
}

```

```js
return repository.head.history.nodes.some((commit: AnyObject) => commit.status);
```

File: source/features/pr-filters.tsx [hunk 2]
```diff
@@ -64,25 +68,35 @@ async function addStatusFilter(): Promise<void> {
...
 	if (!hasCI) {
 		return;
 	}
 
 	// Copy existing element and adapt its content
 	const statusFilter = reviewsFilter.cloneNode(true);
-	const dropdown = select('.select-menu-list', statusFilter)!;
+	statusFilter.id = '';
+
+	select('summary', statusFilter)!.firstChild!.textContent = 'Status '; // Only replace text node, keep caret
```
Comment (fregante) [inline source/features/pr-filters.tsx]: I think `\u00A0` avoided line breaks in that place. Refer to the git blame

Comment (FloEdelmann) [reply source/features/pr-filters.tsx]: Thanks, good catch!

File: source/features/pr-filters.tsx (hunk 1 above)
Comment (FloEdelmann) [reply source/features/pr-filters.tsx]: I'll have a look at this, so I reverted the cache commit to get this PR ready to merge.

**Comment by fregante** [comment]
Looks great! Hope to see the speed improvements too

**Comment by fregante** [comment]
I noticed that `status:pending` is kinda broken:

Screen Shot 2020-02-07 at 14 47 39


(not our fault though)

**Comment by fregante** [comment]
If you get a chance to work on this again, let’s change every mention of “Status” to “Checks”

**Pull request merged**